### PR TITLE
Adding small examples in facade files

### PR DESCRIPTION
**Pull request by kiok46**
Adding small examples for each facade.

It's easier for new users to see these small examples and test them to get a feel of how to work with plyer.

Ready to be merged.

### CODE REVIEW

File: plyer/facades/accelerometer.py [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+    >>> from plyer import accelerometer
+    >>> accelerometer.enable()
+
+To disable acceletometer::
+
+    >>> accelerometer.disable()
+
+To get the acceleration::
+
+    >>> accelerometer.acceleration
```
Comment (rafalo1333) [inline plyer/facades/accelerometer.py]: Nice. I would add the returned values print (as in standard console) to show what's going on, like this:

```python
>>> accelerometer.acceleration
[1, -1, 0]
>>>
```

Comment (kiok46) [inline plyer/facades/accelerometer.py]: That's a nice suggestion, thanks 👍

Comment (kiok46) [inline plyer/facades/accelerometer.py]: @rafalo1333 that's a nice suggestion, thanks 👍

File: plyer/facades/camera.py [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+---------------
+
+To take picture::
+
+    >>> from plyer import camera
+    >>> camera.take_picture(filename, on_complete)
+
+Ta take a video::
+
+    >>> camera.take_video(filename, on_complete)
```
Comment (akshayaurora) [inline plyer/facades/camera.py]: same as above, define `on_complete` first

File: plyer/facades/email.py [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+To send an e-mail::
+
+    >>> from plyer import email
+    >>> recipient = 'abc@gmail.com'
+    >>> subject = 'Hi'
+    >>> text = 'This is an example.'
+    >>> create_chooser = False
+    >>> email.send(recipient=recipient, subject=subject, text=text,
+                   create_chooser=create_chooser)
+
```
Comment (akshayaurora) [inline plyer/facades/email.py]: There should be a mention on how to open a e-mail interface too, where the user can enter his own content.

File: plyer/facades/filechooser.py [hunk 4]
```diff
@@ -51,3 +48,8 @@ def choose_dir(self, *args, **kwargs):
         platform and are planning on using unsupported features.
         """
         return self._file_selection_dialog(mode="dir", *args, **kwargs)
+
+    # private
+
+    def _file_selection_dialog(self, **kwargs):
```
Comment (akshayaurora) [inline plyer/facades/filechooser.py]: If you are doing doc changes don't mix it with code changes.

File: plyer/facades/irblaster.py [hunk 5]
```diff
@@ -47,13 +76,18 @@ def transmit(self, frequency, pattern, mode='period'):
...
-        raise NotImplementedError()
-
     def exists(self):
         '''Check if the device has an infrared emitter.
         '''
         return self._exists()
 
+    # private
+
+    def _get_frequencies(self):
```
Comment (akshayaurora) [inline plyer/facades/irblaster.py]: No code changes in a doc pr!

File: plyer/facades/orientation.py [hunk 6]
```diff
@@ -41,5 +61,13 @@ def set_sensor(self, mode='any'):
         '''
         self._set_sensor(mode=mode)
 
+    # private
+
+    def _set_landscape(self, **kwargs):
```
Comment (akshayaurora) [inline plyer/facades/orientation.py]: ditto,  this is a pr for document changes, do not do code changes in this.

Do separate pr's for code changes.

File: plyer/facades/sms.py [hunk 7]
```diff
@@ -1,3 +1,22 @@
...
+Sms
+====
+
+The :class:`Sms` provides access to sending Sms from your device.
+
+Simple Examples
+---------------
+
+To send sms::
+
```
Comment (akshayaurora) [inline plyer/facades/sms.py]: would be nice if you could mention he permissions needed for every such example too.

**Comment by kiok46** [comment]
@akshayaurora  Thanks, will do the changes 👍

**Comment by yaki29** [comment]
@kiok46 Good one, it already has started helping starters like me ... Thanks

**Comment by yaki29** [comment]
@kiok46 Good one, it already has started helping starters like me ... Thanks

**Comment by akshayaurora** [comment]
@kiok46 thanks. Nice work.
